NIPTool/crud/find/plots/statistics_plot_data.py

```diff
@@ -0,0 +1,52 @@
+from NIPTool.adapter import NiptAdapter
+from NIPTool.crud.find import find
+
+
+def get_last_batches(adapter: NiptAdapter, nr: int) -> list:
+    """geting the <nr> last batches based on SequencingDate"""
+
+    batch_sort_aggregation = [{"$sort": {"SequencingDate": -1}}]
+    sorted_batches = list(find.batch_aggregate(pipe=batch_sort_aggregation, adapter=adapter))
```

Comment:
Suggest to create a function (or modifying existing one) to just do a find with sort and then return a list of batches. That would make things more explicit than working with aggregations.
Something like

```python
def get_batches(adapter, max_result: Optional[int] = None) -> List[Batch]:
    batches: pymongo.Cursor = adapter.db.batches.find().sort({"SequencingDate":-1})
    if max_result:
        batches.limit(max_result)
    return parse_obj_as(blabla)
```

Reply:
👍